### DISCUSSION TITLE
Right-to-left (RTL) language support

### DISCUSSION THREAD

**Discussion by tomisme** [athensresearch/athens#1322] [Feature Requests]
**Linked Discussion: https://github.com/athensresearch/athens/discussions/935**

Questions:

 - Do we mark blocks/pages etc. as being in 'RTL' mode or assume based on the string content?

Resources:

 - https://www.w3.org/International/questions/qa-html-dir

**Comment by jefftangx** [comment]
If we can derive programmatically from Unicode then probably that.

**Comment by mr-islam** [comment]
Just wanted to leave a ping on this issue and show that there interest is for this! 

Clearly Athens is still working towards full release so it's understandable for i18n to take a backseat. I18n entails fonts, sizing, translations and a lot of stuff.

But even simple RTL support (ie. just the document direction flipped) makes a world of difference for these languages, and would open Athens for a lot of people ❤️

**Comment by Hashemian01** [comment]
I believe Athens Research has the potential to deliver an authentic bi-directional RTL/LTR experience as each block can have its own direction property

**Comment by esm7** [comment]
I didn't dig very deep but noticed the repo depends on CodeMirror. If CodeMirror is heavily used as the editor, maybe it's possible to turn on its built-in RTL support and continue from there, similarly to [Obsidian's RTL plugin](https://github.com/esm7/obsidian-rtl).
